src/Client.ts

```diff
@@ -562,76 +557,6 @@ export default class Client {
       mapper
     )
   }
-
-  async signBytes(bytes: Uint8Array): Promise<Signature> {
```

Comment:
I see, this is why the Keystore `sign` function was so generic. The goal was to push this already super generic function into the Keystore API?

Reply:
Actually, this code is totally unused.

The reason why we need the generic sign function in the Keystore is [here](https://github.com/xmtp/xmtp-js/blob/main/src/conversations/Conversation.ts#L493). For V2 messages, the protocol expects the signature to be against the digest of a concatenation of the `payload` and `headerBytes`. Shipping both input values over the wire to the Keystore feels unnecessary, and adding any additional context or metadata would not be backwards compatible.

Reply:
Ah okay that makes sense